Preston_Kellen/Assignments/Dojo_Survey_Validate/dojosurvey.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, render_template, request, redirect, session, flash
+app = Flask(__name__)
+app.secret_key = 'ThisIsSecret'
+
+@app.route('/')
+def index():
+    return render_template("home.html")
+
+@app.route('/users', methods=['POST'])
+def create_user():
+    if len(request.form['name']) < 1:
+        flash("Please enter your name!")
+        return redirect('/')
+    else:
+        flash("Success! Your name is {}".format(request.form['name']))
+    session['name'] = request.form['name']
+    session['location'] = request.form['location']
+    session['language'] = request.form['language']
+    session['comment'] = request.form['comment']
+
+    return redirect('/')
+
+@app.route('/show')
+def show_user():
+    return render_template('user.html')
+
+app.run(debug=True)
```

Comment:
Missing validations on the other 3 fields.  With Dropdown inputs if there is a default option then no need to validate.   Default being the 1st option in your code.  However, you may not always want to leave a default in case you want to make sure the user doesn't get lazy and just skip over it.  In that case you can just put in an empty set of option tags as the first option without any value